test/fixtures/nested.yml

```diff
@@ -14,8 +14,8 @@ sources:
   src:
     build:
       source:
-        context:
-          name: test/fixtures/nested
+        path: test/fixtures/nested
+        context: {}
```

Comment:
Just to note, because of how things get unmarhalled, we need to include *something* in this context struct in order for it to be preserved (or as I've done here with the explicit `{}`